packages/core/src/constants/other.ts

```diff
@@ -1,3 +1,22 @@
+import * as os from 'os';
+import * as path from 'path';
+
+export const STATUS_NOT_EXECUTE = 'not execute';
+export const STATUS_SUCCESS = 'success';
+export const STATUS_FAILURE = 'failure';
+export const STATUS_RUNNING = 'running';
+
+export const DATA = 'data';
+export const MODEL = 'model';
+export const EVALUATE = 'evaluate';
+export const DEPLOYMENT = 'deployment';
+export const MERGE = 'merge';
+export const MODELTOSAVE = 'modeltosave';
+export const ORIGINDATA = 'origindata';
```

Comment:
We have move the above constants into the enum `OutputType` at #185, IMO it's good to extend it here.